config/clirr-exclude.yml

```diff
@@ -5,375 +5,3 @@ differenceTypes:
 - 7011  # (Method Added): className, method
```

Comment:
Yeah, that's what I'd like to solve if possible.  It's quite nice to have a report showing all the added classes and methods.  For a patch release, we don't want to make any API changes, and including this would show if we did.  For minor releases, it's a nice summary of all the API changes that were made.